src/main/javascript/index.js

```diff
@@ -3,7 +3,10 @@
 var BytePushers = {};
 BytePushers.opp = require('bytepushers-js-oop');
 BytePushers.Pick3LotteryWebScrapingService = require('./software.bytepushers.pick3.lottery.web.Pick3LotteryWebScrapingService');
+BytePushers.BaseScraper = require('./software.bytepushers.pick3.lottery.web.BaseScraper');
```

Comment:
Please rename all references (file, variable, and class declaration) to BaseWebScraper.

Reply:
Fixed. @pouncilt